package.json

```
@@ -87,7 +87,7 @@
     "cookie-parser": "^1.4.5",
     "dgraph-js": "^20.11.0",
     "dotenv": "^10.0.0",
-    "ethers": "5.4.7",
+    "ethers": "^5.6.1",
```

```suggestion
    "ethers": "5.4.7",
```
https://github.com/energywebfoundation/iam-client-lib/blob/45db11a8679d1d0af31a5d5a9e84363fcebf1a1d/package.json#L85